src/FrameBuffer.cpp

```diff
@@ -364,9 +365,18 @@ CachedTexture * FrameBuffer::_getSubTexture(u32 _t)
 	if (y0 + copyHeight > m_pTexture->realHeight)
 		copyHeight = m_pTexture->realHeight - y0;
 
-	glBindFramebuffer(GL_READ_FRAMEBUFFER, m_FBO);
+	if (m_pTexture->frameBufferTexture == CachedTexture::fbMultiSample)
+	{
+		resolveMultisampledTexture(true);
+		glBindFramebuffer(GL_READ_FRAMEBUFFER, m_resolveFBO);
+	}
+	else{
+		glBindFramebuffer(GL_READ_FRAMEBUFFER, m_FBO);
+	}
+
```

Comment:
It's possible, I'm getting GL errors when this is attempted. Once I started blitting the non-multisampled one, the issues went away.